docs/EN_US/HPCCDataHandling/DH-Mods/DH-Mod1.xml

```diff
@@ -34,6 +34,74 @@
     that server to enable file sprays and desprays.</para>
   </sect1>
 
+  <sect1 id="FileMovement">
+    <title>File Movement</title>
+
+    <para>The ESP file service can request whether an import (spray), despray,
+    or copy command is a <emphasis role="bold">push</emphasis> or a <emphasis
+    role="bold">pull</emphasis> command.</para>
+
+    <para>These processes run as DFU workunits and communicate with service
+    components, either running as dynamically launched ftslave processes (the
+    default behavior on Bare-Metal deployments), or as tasks running on a
+    dafilesrv service (the only option on a Containerized deployment).</para>
+
+    <itemizedlist>
+      <listitem>
+        <para><emphasis role="bold">pull</emphasis> means that the commands
+        will notionally run per target part and pull from the source.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="bold">push</emphasis> means the command will
```

Comment:
picky: "the commands" (.. and will then match the sentence structure on line 51)